docs/contributing.md

```
@@ -0,0 +1,112 @@
+---
+icon: lucide/code
+---
+
+# Contributing
+
+## Architecture
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)